pkgs/development/compilers/crystal/default.nix

```diff
@@ -53,6 +55,8 @@ let
         tar --strip-components=1 -C $out -xf ${src}
         patchShebangs $out/bin/crystal
       '';
+
+      broken = lib.versionOlder version "1.2.0" && isAarch64Darwin;
```

Comment:
I don't think that's where `broken` goes -- 99% sure it only does things when inside the `meta` attribute, but I could be wrong.

Reply:
Good catch, thanks.